web/src/api/users.ts

```diff
@@ -52,4 +53,16 @@ const fetchRoot = (): Promise<RootUser> => get("/api/users/root");
  */
 const updateRoot = (changes: Partial<RootUser>) => patch("/api/users/root", changes);
 
-export { fetchFirstUser, updateFirstUser, removeFirstUser, fetchRoot, updateRoot };
+/**
+ * Adds the password check result.
```

Comment:
NP: I'd say _Performs a password chec_, _Check strength of given password_, or so.